wtforms/fields/core.py

```diff
@@ -154,6 +157,14 @@ def __call__(self, **kwargs):
         """
         return self.meta.render_field(self, kwargs)
 
+    @classmethod
```

Comment:
classmethod decorator needed to be able to perform a check even when instantiating an unbounded Field (access through field_class)